Sources/Fuzzilli/Modules/Storage.swift

```diff
@@ -66,6 +77,33 @@ public class Storage: Module {
             self.storeProgram(code, to: fileURL)
         }
 
+        if fuzzer.config.diagnostics {
+            fuzzer.registerEventListener(for: fuzzer.events.DiagnosticsEvent) { ev in
+                switch ev.name {
+                    case "REPRLFail":
```

Comment:
This can be more generic, no? I.e. just use name + timestamp as filename and put all of these into a dir named e.g. `diagnostics`.